h2quic/client_test.go

```diff
@@ -4,97 +4,102 @@ import (
 	"bytes"
 	"compress/gzip"
 	"errors"
+	"net"
 	"net/http"
 
 	"golang.org/x/net/http2"
 	"golang.org/x/net/http2/hpack"
 
+	quic "github.com/lucas-clemente/quic-go"
```

Comment:
Can you please clarify? `quic` is needed because we're implementing mock interfaces, e.g. for the `Session`, in the `h2quic` tests. How do you intent to get rid of importing `quic` then?